pkg/spdk/server.go

```diff
@@ -1365,12 +1365,18 @@ func (s *Server) ReplicaBackupStatus(ctx context.Context, req *spdkrpc.BackupSta
 		return nil, grpcstatus.Errorf(grpccodes.NotFound, "cannot find backup %v", req.Backup)
 	}
 
+	replicaAddress := ""
+	if backup.replica != nil {
+		replicaAddress = fmt.Sprintf("tcp://%s:%d", backup.replica.IP, backup.replica.PortStart)
```

Comment:
It's better to use a function like `replica.GetAddress()` rather than directly accessing the replica variables (without locking).

Reply:
Updated. Thank you.